### ISSUE TITLE
foundry frozen bc of gurps plugin

### ISSUE THREAD

**Issue by josmardias** [crnormand/gurps#1321]
I have no idea why but I had to downgrade to last version that worked for me (0.13.4).
Foundry just hangs after 2~3 online players rolling dices.
It takes a while, but not much (less then a minute).

**Comment by crnormand** [comment]
Hmm... does the developer console show anything?   (Press F12 to see it)

**Comment by crnormand** [comment]
Some change occurred in a recent version of Foundry that is causing an "emit" "storm".    If we try to emit data... it is getting bounced around and around.   I disabled that little used feature for now and will investigate.

V0.13.9 seems to work.

**Comment by josmardias** [comment]
It started to "crash" in the middle of our RPG session so I couldn't debug a lot, I just tried disabling/downgrading stuff. After some quick tests I figure that downgrading this plugin to the last working version (0.13.4) did the trick (while keeping the rest upgraded/enabled as usual).

Not sure if it relates to the event emit thing you mentioned but to make it crash I needed people online rolling dices (using the skill buttons IIRC). Just connecting to foundryvtt wasn't enough. The behavior was to slowly feel more laggy with players complaining that their rolls stopped working (nothing shown on the message log) and at some point foundryvtt itself wouldn't answer inputs on the host and finally freezing the host computer.

I tried the console, it didn't show anything meaningful.
